src/frontend/src/eth/components/wallet-connect/WalletConnectSession.svelte

```diff
@@ -330,13 +331,11 @@
 
 		close();
 	};
+
```

Comment:
Just thinking at it now, would it be interesting to add a `onDestroy(() => walletConnectPaired.set(false));` just in case?

Reply:
yes, definitely